.travis.yml

```diff
@@ -5,10 +5,10 @@ matrix:
       env: 'SIMULATOR="name=iPad Pro (12.9-inch) (2nd generation),OS=11.2"'
     - osx_image: xcode9.2
       env: 'SIMULATOR="name=iPhone 7 Plus,OS=10.3.1"'
+    - osx_image: xcode9.2
```

Comment:
I think we should just delete this configuration, it doesn't add value to have a second iOS 10.3.1 configuration IMO.